src/Reflection/ReflectionClassConstant.php

```diff
@@ -91,6 +91,16 @@ public function getValue()
     }
 
     /**
+     * Get the value represented as a string.
+     *
+     * @return string
+     */
+    public function getValueAsString() : string
```

Comment:
What's the advantage of this API? Can't the consumer of `getValue()` call a `var_export()` on it manually?